packages/jest/src/__tests__/matchers.test.tsx

```diff
@@ -1,57 +1,88 @@
 
-  expect(getByText('hello world')).toHaveCompiledCss({
-    fontSize: '12px',
-    color: 'blue',
+    expect(getByText('hello world')).not.toHaveCompiledCss('color', 'blue');
   });
-});
 
-it('should detect single missing styles', () => {
-  const { getByText } = render(<div css={{ fontSize: '12px', color: 'blue' }}>hello world</div>);
+  it('should detect multiple styles', () => {
+    const { getByText } = render(<div css={{ fontSize: '12px', color: 'blue' }}>hello world</div>);
 
-  expect(getByText('hello world')).not.toHaveCompiledCss({
-    zindex: '9999',
+    expect(getByText('hello world')).toHaveCompiledCss({
+      fontSize: '12px',
+      color: 'blue',
+    });
   });
-});
 
-it('should detect multiple missing styles', () => {
-  const { getByText } = render(<div css={{ fontSize: '12px', color: 'blue' }}>hello world</div>);
+  it('should detect single missing styles', () => {
+    const { getByText } = render(<div css={{ fontSize: '12px', color: 'blue' }}>hello world</div>);
 
-  expect(getByText('hello world')).not.toHaveCompiledCss({
-    backgroundColor: 'yellow',
-    zindex: '9999',
+    expect(getByText('hello world')).not.toHaveCompiledCss({
+      zindex: '9999',
+    });
   });
-});
 
-it('should detect evaluated rule from array styles', () => {
-  const base = { fontSize: 12 };
-  const next = ` font-size: 15px; `;
+  it('should detect multiple missing styles', () => {
+    const { getByText } = render(<div css={{ fontSize: '12px', color: 'blue' }}>hello world</div>);
+
+    expect(getByText('hello world')).not.toHaveCompiledCss({
+      backgroundColor: 'yellow',
+      zindex: '9999',
+    });
+  });
+
+  it('should detect evaluated rule from array styles', () => {
+    const base = { fontSize: 12 };
+    const next = ` font-size: 15px; `;
+
+    const { getByText } = render(<div css={[base, next]}>hello world</div>);
+    expect(getByText('hello world')).toHaveCompiledCss('font-size', '15px');
+    expect(getByText('hello world')).toHaveCompiledCss('font-size', '12px');
+  });
+
+  it('should find styles composed from multiple sources', () => {
+    const StyledDiv = styled.div`
+      font-size: 12px;
+    `;
+
+    const { getByText } = render(<StyledDiv css={{ fontSize: 14 }}>Hello world</StyledDiv>);
 
-  const { getByText } = render(<div css={[base, next]}>hello world</div>);
-  expect(getByText('hello world')).toHaveCompiledCss('font-size', '15px');
-  expect(getByText('hello world')).toHaveCompiledCss('font-size', '12px');
+    expect(getByText('Hello world')).toHaveCompiledCss('font-size', '12px');
+    expect(getByText('Hello world')).toHaveCompiledCss('font-size', '14px');
+  });
+
+  it('should find multiple styles composed from multiple sources', () => {
+    const StyledDiv = styled.div`
+      color: blue;
+      background-color: red;
+    `;
+
+    const { getByText } = render(<StyledDiv css={{ color: 'blue' }}>Hello world</StyledDiv>);
```

Comment:
can we change this to `color: red` so its different to the initial StyledDiv?

Reply:
yeah might make more sense to change the color on line 77. Good point!

Reply:
done 👍